data/json/mapgen/wind_turbine.json

```diff
@@ -0,0 +1,223 @@
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "           ###          ",
+        "          ## ##         ",
+        "          #I  #         ",
+        "          ##H##         ",
+        "           ###          ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": {
+        " ": "t_open_air",
+        "H": "t_ladder_up_down",
+        "=": "t_metal_floor_no_roof",
+        "I": "t_metal_floor_no_roof",
+        "#": "t_wall_metal"
+      },
+      "furniture": { "I": "f_electrical_conduit" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "wind_turbine_pillar_platform",
+    "object": {
+      "fill_ter": "t_open_air",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "           ###          ",
+        "          ##=##         ",
+        "          #I==#         ",
+        "          ##H##         ",
+        "           ###          ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": {
+        " ": "t_open_air",
+        "H": "t_ladder_up_down",
+        "=": "t_metal_floor_no_roof",
+        "I": "t_metal_floor_no_roof",
+        "#": "t_wall_metal"
+      },
+      "furniture": { "I": "f_electrical_conduit" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "wind_turbine_nacelle",
+    "object": {
+      "fill_ter": "t_open_air",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "            +           ",
+        "           +++          ",
+        "          ++|++         ",
+        "          ++|++         ",
+        "          ##|##         ",
+        "          #&&&#         ",
+        "          #E|=#         ",
+        "          #I|=#         ",
+        "          #H&&#         ",
```

Comment:
Should the ladder be moved over (and likewise above - and perhaps below; move it into the space opposite the switchgear) so that the maintenance techs would have room to move?

Reply:
Yeah good thinking. And thanks, this model just didn't feel right without the wiring and wind vane!

Reply:
When wist gets to doing portal labs, the conduit furniture could be helpful (also possibly in Exodii and Robofac areas).